internal/commands/load_plugins.go

```diff
@@ -0,0 +1,37 @@
+package commands
+
+import (
+	"context"
+	"fmt"
+	"github.com/instrumenta/conftest/plugin"
+	"github.com/spf13/cobra"
+)
+
+func loadPlugins(ctx context.Context) ([]*cobra.Command, error) {
+	plugins, err := plugin.FindPlugins()
+	if err != nil {
+		return nil, fmt.Errorf("loading plugins: %v", err)
```

Comment:
what's the rationale of not using `%w` pattern just like we applied before to the rest of the code?

Reply:
My bad, I default to `%v` so I just missed it.